kpconv_torch/utils/config.py

```diff
@@ -21,12 +23,23 @@ class Config:
     Class containing the parameters you want to modify for this dataset
     """
 
```

Comment:
Ajout de paramètres à la classe Config pour accueillir les options de la CLI

Reply:
As written before, I think we should reconsider this.